webview-ui/src/components/chat/ChatRow.tsx

```diff
@@ -1043,28 +1043,24 @@ export const ChatRowContent = ({
 								<div style={{ display: "flex", alignItems: "center", gap: "10px", flexGrow: 1 }}>
 									{icon}
 									{title}
-									{
-										// kilocode_change start
-										!cost && usageMissing && (
-											<StandardTooltip content={t("kilocode:pricing.costUnknownDescription")}>
-												<VSCodeBadge className="whitespace-nowrap">
-													<span className="codicon codicon-warning pr-1"></span>
-													{t("kilocode:pricing.costUnknown")}
-												</VSCodeBadge>
-											</StandardTooltip>
-										)
-										// kilocode_change end
-									}
-									<VSCodeBadge
-										style={{ opacity: cost !== null && cost !== undefined && cost > 0 ? 1 : 0 }}>
-										${Number(cost || 0)?.toFixed(4)}
-									</VSCodeBadge>
```

Comment:
this was outside of the kilocode change; did you remove this intentionally?

Reply:
yes, that's the fix

Reply:
so did we forget to mark this? or did Roo also add this?